src/sentry/api/endpoints/client_state.py

```diff
@@ -18,6 +18,31 @@
 }
 
 
+class ClientStateListEndpoint(OrganizationEndpoint):
+    private = True
+
+    def __init__(self, **options) -> None:
+        cluster_key = getattr(settings, "SENTRY_CLIENT_STATE_REDIS_CLUSTER", "default")
+        self.client = redis.redis_clusters.get(cluster_key)
+        super().__init__(**options)
+
+    def get_key(self, organization, category, user):
+        scope = STATE_CATEGORIES[category]["scope"]
+        if scope == "member":
+            return f"client-state:{category}:{organization}:{user}"
+        elif scope == "org":
+            return f"client-state:{category}:{organization}"
+
+    def get(self, request: Request, organization) -> Response:
+        result = {}
+
+        for category in STATE_CATEGORIES:
+            key = self.get_key(organization.slug, category, request.user)
+            value = self.client.get(key)
+            result[category] = json.loads(value)
```

Comment:
@Neo-Zhixing Could this line bork if `value` is `None`?

Reply:
Good point. Fixed and added tests.